app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationController < ActionController::Base
-
+  protect_from_forgery with: :null_session
```

Comment:
I think this is common for API's

WDYT to do this only in API controllers?

Reply:
I understand, but the base application controller is shared by the api and web controllers, that's why it's there.